tests/e2e/pages/aelin/create-pool.js

```diff
@@ -139,15 +214,55 @@ export default class CreatePoolPage extends Page {
   assignPoolPrivacy(privacy) {
     switch (privacy) {
       case PoolPrivacy.PUBLIC:
-        return this.getPrivacyButton(privacy.toLowerCase()).click()
+        return this.getPrivacyLabeledRadioButton(privacy).click()
       case PoolPrivacy.PRIVATE:
         // Private pool logic
         break
       case PoolPrivacy.NFT:
         // NFT pool logic
         break
       default:
-        return this.getPrivacyButton(privacy.toLowerCase()).click()
+        return this.getPrivacyLabeledRadioButton(privacy).click()
+    }
+  }
+
+  assertNoCapCheckBox(checked) {
```

Comment:
All assertions should be done in the spec file. Leave this for simple getters and pool logic methods

Reply:
Strange requirement. It will lead to a lot of repetitive checks and bloated spec files. What's wrong with aggregation of asserts in util functions like this one?

Reply:
Now it looks pretty clean and makes sense. But as this gets bigger and more tests are added, this might get out of control will too many utils/helpers, and making the level of abstraction too high. Anyway this is not critical, we could leave like that and see if in the future we need a refactor